src/scene-entry-manager.js

```diff
@@ -213,6 +214,28 @@ export default class SceneEntryManager {
       spawnMediaInfrontOfPlayer(e.detail, contentOrigin);
     });
 
+    this.scene.addEventListener("object_pinned", e => {
```

Comment:
as per suggestion above in `pinned` component, I would just listen for the `pinned` event and then use `e.target` as `el`